### DISCUSSION TITLE
Add a badge to the `README` with the ktfmt channel in Kotlinlang

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [facebookincubator/ktfmt#179] [Ideas]
Like a lot of Kotlin utilities, there is a channel for `ktfmt` in the kotlinlang Slack, should be great to add it to the `README`:

https://kotlinlang.slack.com/archives/C01GZCU0QNB

**Comment by cgrushko** [comment]
Ah, very cool :) let me get an invite, I'd like to see what's in there.

**Reply by JavierSegoviaCordoba** [reply]
https://surveys.jetbrains.com/s3/kotlin-slack-sign-up

**Comment by hick209** [comment]
We have added the Koltin slack channel [here](https://facebook.github.io/ktfmt/), but I think it would be valuable to add that in the `README` as well and with the link for joining.

@JavierSegoviaCordoba, do you want to submit a quick PR adding this?

**Reply by JavierSegoviaCordoba** [reply]
Done: https://github.com/facebook/ktfmt/pull/448
